packages/react-components/react-field/src/components/Field/useFieldStyles.ts

```diff
@@ -19,16 +19,23 @@ const useRootStyles = makeStyles({
   base: {
     display: 'grid',
     gridAutoFlow: 'row',
+    gridTemplateColumns: 'auto 1fr',
     justifyItems: 'start',
   },
 
   horizontal: {
     gridTemplateRows: 'auto auto auto auto',
-    gridTemplateColumns: '1fr 2fr',
+    gridTemplateColumns: '1fr auto 2fr',
```

Comment:
Just thought of a potential issue here. Since it's using `auto` for the validation message column and the `fr` unit for the other columns, it might cause misalignment between fields that do and don't have a validation message (or movement when a validation message appears).

For example, if the fields are 312px wide, then with no validation icon, there's 312px to split between the other columns, making the label column 104px wide. But with a validation icon, the second column will be 12px, leaving 300px to split, making the label column 100px wide.

Test case:
```jsx
<div style={{ display: 'grid', width: '312px' }}>
  <InputField label="First Name" orientation="horizontal" />
  <InputField label="Last Name" orientation="horizontal" validationState="error" validationMessage="test" />
</div>
```

Probably the best way to fix it is to change the columns to make the label a percentage of the grid width, not the free space:
```suggestion
    gridTemplateColumns: '33% auto 1fr',
```

Reply:
oooo, good catch 👍